src/Workspaces/Core/Portable/Differencing/Match.cs

```diff
@@ -12,10 +12,9 @@ public sealed partial class Match<TNode>
     {
         private const double ExactMatchDistance = 0.0;
         private const double EpsilonDistance = 0.00001;
-        private const double MatchingDistance1 = 0.5;
-        private const double MatchingDistance2 = 1.0;
-        private const double MatchingDistance3 = 1.5;
-        private const double MaxDistance = 2.0;
+        private const double MatchingDistance1 = 0.25;
+        private const double MatchingDistance2 = 0.5;
+        private const double MaxDistance = 1.0;
```

Comment:
this change worries me.  @tmat Do we have enough test coverage to have confidence this won't result in poorer matches for existing code?